packages/yourloops/test/integration/mock/team.api.mock.ts

```diff
@@ -186,6 +198,8 @@ export const iTeamOne: ITeam = {
   members: null
 }
 
+export const availableTeams = [teamOne, teamTwo, teamThree, teamPrivate]
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏
```suggestion
export const TEAMS_AVAILABLE = [teamOne, teamTwo, teamThree, teamPrivate]
```

Reply:
Can we agree on `AVAILABLE_TEAMS`?

Reply:
Yes we can